### DISCUSSION TITLE
Welcome to FileConverter Discussions!

### DISCUSSION THREAD

**Discussion by Tichau** [Tichau/FileConverter#135] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together ❤.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Liverrrrr** [comment]
Hey there, I just went through way too much time looking through ALL the open and closed issues, but of course i probably knew I wouldn't find the answer there; too stupid a question. I downloaded the source code zip and extracted and blah blah blah but how do I actually make it operable? I see in the installer folder there is a Installer.wixproj file and a batch file that doesn't seem to do anything when i open it, along with a product.wxs, none of which I can really understand when opened. Please tell me how to build this thing 😅

**Reply by xaryangupta** [reply]
Same doubt, I have the VS Community app. Clicked on .sln file and tried to build it but nah, pls update it
